pages/accountLists/[accountListId]/tools/fixMailingAddresses.page.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react';
+import { useTranslation } from 'react-i18next';
+import Head from 'next/head';
+import FixMailingAddresses from '../../../../src/components/Tool/FixMailingAddresses/FixMailingAddresses';
+
+const fixMailingAddresses = (): ReactElement => {
```

Comment:
I think `React.FC<MyProps>` is usually better and you get the children prop for free.
```suggestion
const fixMailingAddresses: React.FC = () => {
```